.github/workflows/aws-staging-kubernetes.yaml

```diff
@@ -0,0 +1,110 @@
+on:
+  push:
+    branches:
+      - dev-k8s
+
+name: Deploy to AWS Staging (K8s)
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: staging
+    env:
+      AWS_REGION: eu-central-1
+      CLUSTER_NAME: maker-staging
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+
+    - name: Login to Amazon ECR
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Extract commit hash
+      id: vars
+      shell: bash
+      run: |
+        echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+    - name: Build, tag, and push image to Amazon ECR
+      id: build-image
+      env:
+        SHA_TAG: ${{ steps.vars.outputs.sha_short }}
+        LATEST_TAG: latest
+        ENVIRONMENT_TAG: staging
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+      run: |  
+        # Build Docker containers and
+        # push them to ECR:
+        # - chainlog-ui
+        docker pull $ECR_REGISTRY/chainlog-ui-staging:$LATEST_TAG || true
+        docker build -t chainlog-ui-staging \
+                     -t $ECR_REGISTRY/chainlog-ui-staging:$SHA_TAG \
+                     -t $ECR_REGISTRY/chainlog-ui-staging:$LATEST_TAG \
+                     -t $ECR_REGISTRY/chainlog-ui-staging:$ENVIRONMENT_TAG \
+                     -f Dockerfile.ui \
+                     .
+        docker push $ECR_REGISTRY/chainlog-ui-staging --all-tags
+        # - chainlog-checksum
+        docker pull $ECR_REGISTRY/chainlog-checksum-staging:$LATEST_TAG || true
+        docker build -t chainlog-checksum-staging \
+                     -t $ECR_REGISTRY/chainlog-checksum-staging:$SHA_TAG \
+                     -t $ECR_REGISTRY/chainlog-checksum-staging:$LATEST_TAG \
+                     -t $ECR_REGISTRY/chainlog-checksum-staging:$ENVIRONMENT_TAG \
+                     -f Dockerfile.checksum \
+                     .
+        docker push $ECR_REGISTRY/chainlog-checksum-staging --all-tags
```

Comment:
What does this do if it's not getting deployed?

Reply:
@eskp didn't get the question, could you please give more details?

Reply:
Just wondering what the checksum container does cos you're not deploying it

Reply:
@eskp don't know I just followed exactly what CICD does for ECS stack. Possible some other component/human pulls that image